packages/ckeditor5-list/tests/listproperties/listpropertiesui.js

```diff
@@ -205,6 +205,16 @@ describe( 'ListPropertiesUI', () => {
 					sinon.assert.calledOnce( editor.editing.view.focus );
 					sinon.assert.callOrder( editor.execute, editor.editing.view.focus );
 				} );
+
+				describe( 'arrow', () => {
+					it( 'should have a #label', () => {
+						expect( mainButtonView.arrowView.label ).to.equal( 'Bulleted List' );
+					} );
+
+					it( 'should have a #tooltip based on a label', () => {
+						expect( mainButtonView.arrowView.tooltip ).to.be.true;
+					} );
+				} );
```

Comment:
Obsolete per my comment in [codeblockui.js](https://github.com/ckeditor/ckeditor5/pull/11859/files?show-viewed-files=true&file-filters%5B%5D=#diff-a75a67319b38c5dc7c27b30eeaff970892cdd73e01b28920a569d161458ea8c7).